lib/rules/padded-blocks.js

```diff
@@ -96,9 +96,12 @@ module.exports = function(context) {
                 if (!blockHasTopPadding) {
                     context.report(node, ALWAYS_MESSAGE);
                 }
-
                 if (!blockHasBottomPadding) {
-                    context.report(node, node.loc.end, ALWAYS_MESSAGE);
+                    context.report({
+                        node: node,
+                        loc: {line: node.loc.end.line, column: node.loc.end.column - 1 },
```

Comment:
Doesn't this just mean there's an off-by-one in the parser? Why would we have to do this offset?